charts/values.yaml

```diff
@@ -97,7 +97,7 @@ jobTemplate:
   image:
     registry: ghcr.io
     repository: cloudtty/cloudshell
-    tag: "latest"
+    tag: "v0.1.0"
```

Comment:
hmm... in `Makefile`, this image is also `latest`, could you please fix it as well ?